build.gradle

```diff
@@ -37,9 +37,9 @@ dependencies {
   compile gradleApi()
   compile localGroovy()
 
-  compile 'com.squareup.spoon:spoon-runner:1.1.2'
+  compile 'com.squareup.spoon:spoon-runner:1.1.5'
 
-  compile 'com.android.tools.build:gradle:0.14.1'
+  compile 'com.android.tools.build:gradle:1.0.0'
```

Comment:
Nope, wasn't that (saw that in the readme).  It's a conflict with the android-unit-test plugin.  Trying to resolve now.